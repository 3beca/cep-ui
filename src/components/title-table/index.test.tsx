import * as React from 'react';
import { render } from '@testing-library/react';
import TitleTable from './index';

test('TitleTable render with the title provided and snapshot', () => {
    const title = 'Table title component';
    const { container, getByLabelText } = render(<TitleTable title={title} />);

    expect(getByLabelText(/table title/i)).toHaveTextContent(title);
    expect(container).toMatchSnapshot();
});
