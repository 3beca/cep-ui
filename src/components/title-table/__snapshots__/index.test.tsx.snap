// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TitleTable render with the title provided and snapshot 1`] = `
<div>
  <div
    aria-label="table title"
    class="makeStyles-tabletitle-1"
  >
    <div
      class="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
    >
      <h6
        class="MuiTypography-root makeStyles-tablename-2 MuiTypography-h6"
      >
        Table title component
      </h6>
    </div>
  </div>
</div>
`;
